commit 2360fc6a704e8dc1f8183a048980052bfe64388b
Author: Sergei Shtylyov <sergei.shtylyov@cogentembedded.com>
Date:   Mon Oct 1 22:57:39 2018 +0300

    dt-bindings: pwm: renesas: pwm-rcar: Document R8A779{7|8}0 bindings
    
    Document the R-Car V3{M|H} (R8A779{7|8}0) SoC in the  Renesas R-Car PWM
    bindings. R8A77970's hardware is a generic R-Car gen3 PWM, while R8A77980
    has an extra error injection register...
    
    Signed-off-by: Sergei Shtylyov <sergei.shtylyov@cogentembedded.com>
    Reviewed-by: Geert Uytterhoeven <geert+renesas@glider.be>
    Reviewed-by: Simon Horman <horms+renesas@verge.net.au>
    Signed-off-by: Thierry Reding <thierry.reding@gmail.com>

diff --git a/Documentation/devicetree/bindings/pwm/renesas,pwm-rcar.txt b/Documentation/devicetree/bindings/pwm/renesas,pwm-rcar.txt
index e1ef6afbe3a7..e68a34e57ff6 100644
--- a/Documentation/devicetree/bindings/pwm/renesas,pwm-rcar.txt
+++ b/Documentation/devicetree/bindings/pwm/renesas,pwm-rcar.txt
@@ -12,6 +12,8 @@ Required Properties:
  - "renesas,pwm-r8a7795": for R-Car H3
  - "renesas,pwm-r8a7796": for R-Car M3-W
  - "renesas,pwm-r8a77965": for R-Car M3-N
+ - "renesas,pwm-r8a77970": for R-Car V3M
+ - "renesas,pwm-r8a77980": for R-Car V3H
  - "renesas,pwm-r8a77990": for R-Car E3
  - "renesas,pwm-r8a77995": for R-Car D3
 - reg: base address and length of the registers block for the PWM.